### ISSUE TITLE
Null resource when trying to perform raw query

### ISSUE THREAD

**Issue by alexbowers-tecmark** [UnionOfRAD/lithium#1210]
I am trying to perform a raw query to set a session variable in mysql; using the following code:

```php
 Products::connection()->read('SET SESSION group_concat_max_len = 1000000;');
```

This previously worked fine, however on the new 1.0.0-RC1 branch, this is now throwing the following error: 

```
Fatal error: Call to a member function columnCount() on null in /var/www/vhosts/project/libraries/lithium/data/source/Database.php on line 834
```

The object being performed on is: 
```
lithium\data\source\database\adapter\pdo\Result Object
(
    [_iterator:protected] => 0
    [_current:protected] => 
    [_valid:protected] => 
    [_key:protected] => 
    [_resource:protected] => 
    [_autoConfig:protected] => Array
        (
            [0] => resource
        )

    [_buffer:protected] => Array
        (
        )

    [_config:protected] => Array
        (
            [resource] => PDOStatement Object
                (
                    [queryString] => SET SESSION group_concat_max_len = 1000000;
                )

            [init] => 1
        )

    [_methodFilters:protected] => Array
        (
        )

)
```

**Comment by davidpersson** [comment]
From which commit or version have you been updating to 1.0.0-rc1?

**Comment by davidpersson** [comment]
Thank you for your bug report. Should be fixed by 3fa37eb, please test and verify.

**Comment by alexbowers-tecmark** [comment]
Seems fine.
